pkg/cmd/cli/cmd/set/env.go

```diff
@@ -273,14 +382,45 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 				c.Env = updateEnv(c.Env, env, remove)
 
 				if list {
+					resolveErrors := map[string][]string{}
```

Comment:
should we remember if any containers hit resolve errors, and do something like this at the end of this func rather than return nil:
```
if resolutionErrorsEncountered {
  errored = append(errored, info)
  return errors.New("error retrieving valueFrom references")
}
```